components/fxa-client/ios/FxAClient/FxAccount.swift

```diff
@@ -116,6 +116,35 @@ class FxAccount: RustFxAccount {
         }
     }
 
+    override func migrateFromSessionToken(sessionToken: String, kSync: String, kXCS: String) -> Bool {
```

Comment:
Can iOS receive a JSON blob of metrics (that they can later report to Telemetry)?

Reply:
Yes they can, I suggest we make that change when they actually want to use it.

Reply:
Okay, I added https://jira.mozilla.com/browse/SYNC-822 so we can add it later